sig-ops/README.md

```diff
@@ -0,0 +1,27 @@
+# SIG: Ops
+
+This Special Interest Group (SIG) main goal is to coordinate Operations (Ops) activities and best practices across Spinnaker Operators.
+* Production Readiness
+* Security Requirements
+* Reference Architecture(s)  
+
+## Meetings
+
+* Regular SIG Meeting
+  * Every other TBD
+  * [Agenda]
+
+## Leadership
+
+* Edgar Magana (emagana](https://github.com/emagana))
```

Comment:
```suggestion
* Edgar Magana [emagana](https://github.com/emagana))
```

Reply:
I have been talking with other teams about it. I am sure after the summit we will be able to get enough members.